### DISCUSSION TITLE
Error fetching node from poolpicker

### DISCUSSION THREAD

**Discussion by DiscoverySDR** [revoxhere/duino-coin#1634] [Q&A]
Hi,

I have several ESP32 / ESP8266 for Duino-Coin mining.

Since last week all the miners have been stopped and the following message appears:
Error fetching node from poolpicker: connection failed

I have tried to configure new ESP8266 from scratch but without success (same error appears).

Please any support would be very appreciated.

Thanks!

**Comment by ppyote** [comment]
Change DNS
